python/paddle/optimizer/momentum.py

```diff
@@ -353,6 +485,89 @@ def _append_optimize_op(self, block, param_and_grad):
 
         return momentum_op
 
+    def _append_optimize_multi_tensor_op(self, target_block,
+                                         parameters_and_grads):
+        assert isinstance(target_block, framework.Block)
+
+        self.grad_dict = {'FP32_LODTensor': [], 'FP16_LODTensor': []}
+        self.lr_dict = {'FP32_LODTensor': [], 'FP16_LODTensor': []}
+
+        if framework.in_dygraph_mode():
```

Comment:
same above, what about static mode?

Reply:
Tks, Multi Tensor has been added to the static mode.